src/main/archetype/ui.frontend/webpack.common.js

```diff
@@ -5,6 +5,7 @@ const webpack                 = require('webpack');
 const MiniCssExtractPlugin    = require("mini-css-extract-plugin");
 const TSConfigPathsPlugin     = require('tsconfig-paths-webpack-plugin');
 const TSLintPlugin            = require('tslint-webpack-plugin');
+const CopyPlugin              = require('copy-webpack-plugin');
```

Comment:
I'd rename the variable to `CopyWebpackPlugin` to be consistent with the CleanWebpackPlugin naming convention?